File: frontend/planta-smart-4.0/src/infrastructure/dtos/response/stream/ProcessoStreamDTO.ts
export interface ProcessoStreamDTO {
  estacao: 'processo';
  status: string;
  numeroOP: number;
  ocupado: boolean;
  aguardando: boolean;
  manual: boolean;
  emergencia: boolean;
  recebidoOp: boolean;
  startOP: boolean;
  finishOP: boolean;
  cancelOP: boolean;
  statusBancada: number;
  supervisorioEstoque: string;
  supervisorioProcesso: string;
  supervisorioMontagem: string;
  supervisorioExpedicao: string;
}   

File: backend/planta-smart-4.0/src/main/java/com/smart/appsa/dto/clp/stream/ProcessoStreamDTO.java
package com.smart.appsa.dto.clp.stream;

import com.smart.appsa.model.enums.StatusEstacao;

public record ProcessoStreamDTO(
        String estacao,
        StatusEstacao status,
        int numeroOP,
        boolean ocupado,
        boolean aguardando,
        boolean manual,
        boolean emergencia,
        boolean recebidoOp,
        boolean startOP,
        boolean finishOP,
        boolean cancelOP,
        int statusBancada) {
}
